src/Extensions/Abstractions/EventSources/EventSourcesEventIds.cs

```diff
@@ -1,11 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-namespace Microsoft.Omex.Extensions.EventSources
+namespace Microsoft.Omex.Extensions.Abstractions.EventSources // diffirent namespace since this class has specific usage
```

Comment:
>diffirent namespace since this [](start = 65, length = 30)

I don't think you want this comment anymore

Reply:
I've wanted to keep it to emphasize the importance of namespace